### PR TITLE
Added serializer, viewset and url for top KVED

### PR DESCRIPTION
**Pull request by IegorSoboliev**
@roman-tiukh please, review if this is ok and then I will add select_related and so on

### CODE REVIEW

File: data_converter/urls.py [hunk 1]
```diff
@@ -45,6 +47,8 @@
 urlpatterns = [
     path('api/stats/', include('stats.urls')),
 
+    path('api/stats/top-kved/', TopKvedsView.as_view(), name='top_kved'),
```
Comment (roman-tiukh) [inline data_converter/urls.py:50]: move this path to stats/urls.py

File: stats/views.py [hunk 2]
```diff
@@ -17,3 +24,11 @@ def get(self, request):
         )
 
         return Response({'days': days}, status=200)
+
+
+class TopKvedsView(generics.ListAPIView):
+    permission_classes = [AllowAny]
```
Comment (roman-tiukh) [inline stats/views.py:30]: why AllowAny?

File: stats/serializers.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    kved = serializers.SerializerMethodField()
+    count_kved = serializers.IntegerField()
+
+    class Meta:
+        model = Kved
+        fields = ['kved', 'count_kved']
+
+    def get_kved(self, kved):
+        kved_info = Kved.objects.get(id=kved['kved_id'])
+        return KvedDetailSerializer(kved_info).data
```
Comment (roman-tiukh) [inline stats/serializers.py:17]:
```suggestion
class TopKvedSerializer(serializers.ModelSerializer):
    kved = serializers.KvedDetailSerializer()
    count_kved = serializers.IntegerField()
    
    class Meta:
        model = CompanyToKved
        fields = ['kved', 'count_kved']
```

File: stats/views.py [hunk 4]
```diff
@@ -17,3 +24,11 @@ def get(self, request):
...
+    queryset = (CompanyToKved.objects.values('kved_id')
+                    .annotate(count_kved=Count('kved_id'))
+                    .order_by('-count_kved')[:10])
+    serializer_class = TopKvedSerializer
```
Comment (roman-tiukh) [inline stats/views.py:34]:
```suggestion
class TopKvedsView(generics.ListAPIView):
    queryset = CompanyToKved.objects.select_related(
        'kved', 'kved__group', 'kved__section', 'kved__division',
    ).annotate(
        count_kved=Count('kved')
    ).order_by('-count_kved')[:10]
    serializer_class = TopKvedSerializer
```

File: stats/views.py (hunk 2 above)
Comment (IegorSoboliev) [reply stats/views.py:30]: maybe this stats is the best attraction for our project)

File: stats/views.py (hunk 4 above)
Comment (IegorSoboliev) [reply stats/views.py:34]: this way we get only 1 as a result of count for every CompanyToKved object.  That is why I have used values()

File: data_converter/urls.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_converter/urls.py:50]: done, but it doesn't work

File: stats/urls.py [hunk 5]
```diff
@@ -1,6 +1,9 @@
 from django.urls import path
+
 from . import views
+from .views import TopKvedsView
 
 urlpatterns = [
-    path('api-usage/me/', views.ApiUsageMeView.as_view())
+    path('api-usage/me/', views.ApiUsageMeView.as_view()),
+    path('api/stats/top-kved/', TopKvedsView.as_view(), name='top_kved'),
```
Comment (roman-tiukh) [inline stats/urls.py:8]: wrong usage
```suggestion
    path('top-kved/', TopKvedsView.as_view(), name='top_kved'),
```

File: stats/views.py (hunk 2 above)
Comment (roman-tiukh) [reply stats/views.py:30]: have you discussed the availability of this view with Lyuba or Evgeniy?

File: data_converter/urls.py (hunk 1 above)
Comment (roman-tiukh) [inline data_converter/urls.py:50]: remove this line

File: stats/views.py (hunk 2 above)
Comment (IegorSoboliev) [reply stats/views.py:30]: not yet)

File: data_converter/urls.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_converter/urls.py:50]: done

File: stats/urls.py (hunk 5 above)
Comment (IegorSoboliev) [reply stats/urls.py:8]: thanks))

**Pull request merged**